src/metabase/query_processor.clj

```diff
@@ -85,10 +88,13 @@
       dev/check-results-format
       limit/limit
       cumulative-ags/handle-cumulative-aggregations
-      implicit-clauses/add-implicit-clauses
       format-rows/format-rows
       results-metadata/record-and-return-metadata!
-      expand-resolve/expand-resolve                    ; ▲▲▲ QUERY EXPANSION POINT  ▲▲▲ All functions *above* will see EXPANDED query during PRE-PROCESSING
+      resolve/resolve-middleware
+      add-dim/add-remapping
+      implicit-clauses/add-implicit-clauses
+      source-table/resolve-source-table-middleware
+      expand/expand-middleware                      ; ▲▲▲ QUERY EXPANSION POINT  ▲▲▲ All functions *above* will see EXPANDED query during PRE-PROCESSING
```

Comment:
this is super nitpicky but can we keep the EOL comments aligned? it doesn't matter but it makes me itchy and I will literally be compelled to fix it next time I run across this namespace